File: frontend/lib/models/service.dart
class Service {
  final int id;
  final String slug;
  final String nameRu;
  final String? descriptionRu;
  final String? iconUrl;
  final bool isActive;
  final int sortOrder;
  
  Service({
    required this.id,
    required this.slug,
    required this.nameRu,
    this.descriptionRu,
    this.iconUrl,
    required this.isActive,
    required this.sortOrder,
  });
  
  factory Service.fromJson(Map<String, dynamic> json) {
    return Service(
      id: json['id'],
      slug: json['slug'],
      nameRu: json['name_ru'],
      descriptionRu: json['description_ru'],
      iconUrl: json['icon_url'],
      isActive: json['is_active'],
      sortOrder: json['sort_order'],
    );
  }
  
  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'slug': slug,
      'name_ru': nameRu,
      'description_ru': descriptionRu,
      'icon_url': iconUrl,
      'is_active': isActive,
      'sort_order': sortOrder,
    };
  }
}

File: backend/app/schemas/service.py
"""
Pydantic schemas for Service
"""
from pydantic import BaseModel, Field
from typing import Optional


class ServiceBase(BaseModel):
    """Базовая схема услуги"""
    slug: str = Field(..., max_length=100)
    name_ru: str = Field(..., max_length=255)
    description_ru: Optional[str] = None
    icon_url: Optional[str] = Field(None, max_length=500)
    is_active: bool = True
    sort_order: int = 0


class ServiceCreate(ServiceBase):
    """Схема создания услуги"""
    pass


class ServiceUpdate(ServiceBase):
    """Схема обновления услуги"""
    slug: Optional[str] = None
    name_ru: Optional[str] = None


class ServiceResponse(ServiceBase):
    """Схема ответа услуги"""
    id: int
    
    class Config:
        from_attributes = True


class ServiceList(BaseModel):
    """Список услуг"""
    items: list[ServiceResponse]
    total: int
